tests/Doctrine/Tests/ORM/Functional/Ticket/GH5742Test.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\ORM\Functional\Ticket;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Doctrine\Tests\OrmFunctionalTestCase;
+
+class GH5742Test extends OrmFunctionalTestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->createSchemaForModels(
+            GH5742Person::class,
+            GH5742Toothbrush::class,
+            GH5742ToothpasteBrand::class
```

Comment:
Aaaah a funny test at last… this is going to be good! :popcorn:

Reply:
I designed it that way particularly with you in mind 😉